templates/kommander.yaml

```
@@ -30,7 +30,7 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.1.29
+    version: 0.1.30
```

Just released - please update again:

```suggestion
    version: 0.1.31
```